adhocteam/Head-Start-TTADP
# Align label and title on total hours and TTA graph

## Description of change
Label and title were not aligned on the total hours and TTA graph widget. In addition, the reason list would sometimes not fill its container.


## How to test

Visit the dashboard page.
Confirm that the date and title line up on the line graph. 
Confirm that the reason list fills its container vertically.

## Issue(s)

## Checklists

[n/a] Meets issue criteria
[n/a ] JIRA ticket status updated
**Code is meaningfully tested**
**Meets accessibility standards (WCAG 2.1 Levels A, AA)**
[n/a] API Documentation updated
[n/a] Boundary diagram updated
[n/a] Logical Data Model updated
[n/a] Architectural Decision Records written for major infrastructure decisions

### Production Deploy


### After merge/deploy
